modules/efk-logging-fluentd-log-rotation.adoc

```diff
@@ -19,23 +19,34 @@ fluentd.log.3  1Mb
 fluentd.log.4  1Mb
 ----
 
+The following example shows the same conditions for Rsyslog.
+
+----
+rsyslog.log     0b
+rsyslog.log.1  1Mb
+rsyslog.log.2  1Mb
+rsyslog.log.3  1Mb
+rsyslog.log.4  1Mb
+----
```

Comment:
The log files for rsyslog is looks like:
-rw-r--r--. 1 root root      467 Jul 17 03:40 logrotate.log
-rw-r--r--. 1 root root     1007 Jul 17 03:25 rsyslog.log
-rw-r--r--. 1 root root 10382396 Jul 17 02:49 rsyslog.log-20190717-1563331501
-rw-r--r--. 1 root root 12649621 Jul 17 03:02 rsyslog.log-20190717-1563332101
-rw-r--r--. 1 root root 11345789 Jul 17 03:11 rsyslog.log-20190717-1563332701
-rw-r--r--. 1 root root 12271294 Jul 17 03:21 rsyslog.log-20190717-1563333302
-rw-r--r--. 1 root root 23122678 Jul 17 03:40 rsyslog.log-20190717-1563333901

Reply:
Thank you. I forgot that I wanted an overly-simple example without the full naming to show the rotation. I'm not sure the fluentd log file names I have are accurate for what the user will see. 
Do you think this will work to show the rotation, or would using real log file names make more sense? 
```
The following example shows logs in a cluster where the maximum log size is 1Mb and four logs should be retained. When the `log`reaches 1Mb, {product-title}
deletes the current `log.4`, renames the each of the Fluentd logs in turn, and creates a new `log`.

----
log     0b
log.1  1Mb
log.2  1Mb
log.3  1Mb
log.4  1Mb
----
```